diff --git a/node_modules/@auto-it/upload-assets/dist/index.js b/node_modules/@auto-it/upload-assets/dist/index.js
index dd2f5b9..e4d6689 100644
--- a/node_modules/@auto-it/upload-assets/dist/index.js
+++ b/node_modules/@auto-it/upload-assets/dist/index.js
@@ -130,6 +130,7 @@ class UploadAssetsPlugin {
             if (Array.isArray(release)) {
                 await Promise.all(release.map(async (r) => {
                     const { data: releaseAsset, } = await auto.git.github.repos.uploadReleaseAsset(Object.assign(Object.assign({}, options), { release_id: r.data.id }));
+                    auto.logger.log.success(`Uploaded asset to package: ${r.data.name}`);
                     assetResponses.push(releaseAsset);
                 }));
             }
